packages/docs-theme/src/tags/css.tsx

```diff
@@ -4,30 +4,32 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import { IKssExample, IKssModifier, IKssPluginData } from "documentalist/dist/client";
+import { IKssModifier, IKssPluginData, ITag } from "documentalist/dist/client";
 import * as React from "react";
+import { DocumentationContextTypes, IDocumentationContext } from "../common/context";
 import { ModifierTable } from "../components/modifierTable";
-import { TagRenderer } from "./";
 
-const MODIFIER_PLACEHOLDER = /\{\{([\.\:]?)modifier\}\}/g;
-const DEFAULT_MODIFIER: IKssModifier = {
-    documentation: "Default",
-    name: "default",
-};
-
-const CssExample: React.SFC<IKssExample> = ({ markup, markupHtml, modifiers, reference }) => (
-    <div>
-        {modifiers.length > 0 ? <ModifierTable modifiers={modifiers} /> : undefined}
-        <div className="docs-example-wrapper" data-reference={reference}>
-            {renderMarkupForModifier(markup, DEFAULT_MODIFIER)}
-            {modifiers.map(mod => renderMarkupForModifier(markup, mod))}
+export const CssExample: React.SFC<ITag> = ({ value }, { getDocsData }: IDocumentationContext) => {
+    const { css } = getDocsData() as IKssPluginData;
+    if (css == null || css[value] == null) {
+        return null;
+    }
+    const { markup, markupHtml, modifiers, reference } = css[value];
+    return (
+        <div>
+            {modifiers.length > 0 ? <ModifierTable modifiers={modifiers} /> : undefined}
+            <div className="docs-example-wrapper" data-reference={reference}>
+                {renderMarkupForModifier(markup, DEFAULT_MODIFIER)}
+                {modifiers.map(mod => renderMarkupForModifier(markup, mod))}
+            </div>
+            <div className="docs-markup" dangerouslySetInnerHTML={{ __html: markupHtml }} />
         </div>
-        <div className="docs-markup" dangerouslySetInnerHTML={{ __html: markupHtml }} />
-    </div>
-);
+    );
+};
+CssExample.contextTypes = DocumentationContextTypes;
+CssExample.displayName = "Docs.CssExample";
```

Comment:
prefer making this a class for consistency, since it's complex enough (it requires a helper render function)